Formula/fcl.rb

```diff
@@ -0,0 +1,30 @@
+class Fcl < Formula
+  desc "Flexible Collision Library"
+  homepage "https://flexible-collision-library.github.io/"
+  url "https://github.com/flexible-collision-library/fcl/archive/0.5.0.tar.gz"
+  sha256 "8e6c19720e77024c1fbff5a912d81e8f28004208864607447bc90a31f18fb41a"
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "flann"
+  depends_on "libccd"
```

Comment:
Could you add `boost` and `octomap` as the dependencies ([boost](https://github.com/flexible-collision-library/fcl/blob/fcl-0.5/CMakeLists.txt#L167), [octomap](https://github.com/flexible-collision-library/fcl/blob/fcl-0.5/CMakeLists.txt#L70-L101))?